pkg/blob/blob.go

```diff
@@ -167,11 +168,29 @@ type DriverOptions struct {
 	SasTokenExpirationMinutes              int
 }
 
+func (option *DriverOptions) AddFlags() {
```

Comment:
if you could add ut for this func, it should improve the test coverage, current num is below 80%

Reply:
please refer to TestDriverOptions_AddFlags